### PR TITLE
Performance: Add Dojo Widget to test suite

### PR DESCRIPTION
**Pull request by geekman-rohit**
This PR adds Dojo Widget (dijit) test to the test suite .
[Link to the documentation dijit/form/Button](http://dojotoolkit.org/reference-guide/1.10/dijit/form/Button.html#dijit-form-button)

There are not many variants I could figure out from the documentation. Though there are combo buttons, I have not added those because similar combo-button with dropdown are available in other frameworks like bootstrap, and they were not added in test suite, hence, it would not have been a fair comaprison.

### CODE REVIEW

File: performance/frameworks/dojo.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+			button += "'></span>" +
+				"<span class='dijitReset dijitToggleButtonIconChar'>●</span>" +
+				"<span class='dijitReset dijitInline dijitButtonText ";
+
+			// Check if Button text is to be displayed
+			if ( options.type ) {
+				button += " dijitDisplayNone";
+			}
+
+			button += " '>cut</span></span></span>" +
```
Comment (arschmitz) [inline performance/frameworks/dojo.js]: you have `cut` hard coded here lets use the icon value or a generic text of `button`

**Comment by geekman-rohit** [comment]
Yes, thanks for pointing out! updated  the PR with a new commit.

**Comment by arschmitz** [comment]
@geekman-rohit your using input type=buttons here instead of button elements can you switch that for consistency with other tests please.

**Comment by geekman-rohit** [comment]
@arschmitz dojo replaces button with the markup generated above, this markup is added by dojo javascript in place of a button element.

**Comment by arschmitz** [comment]
@geekman-rohit ah your right thank you

**Pull request closed without merging**